### ISSUE TITLE
[Feature Request] Add loss functions for classification tasks

### ISSUE THREAD

**Issue by arijitde92** [Avdhesh-Varshney/AI-Code#20]
The repo currently lacks loss functions specifically used for classification tasks in the ML category (`ML -> algorithms -> Losses` folder).

I want to introduce some widely used loss functions for classification tasks like [Cross Entropy](https://en.wikipedia.org/wiki/Cross-entropy) Loss, [Hinge Loss](https://en.wikipedia.org/wiki/Hinge_loss) and [Kullback Leibler Divergence](https://en.wikipedia.org/wiki/Kullback%E2%80%93Leibler_divergence) Loss.

I plan to write the loss functions from scratch (using [numpy](https://numpy.org/)) and also write test cases to test the code.

@Avdhesh-Varshney Please assign me this issue if you deem it a suitable addition to the repo.
Also let me know your suggestions and feedback regarding this issue as well.

Thanks.

**Comment by Avdhesh-Varshney** [comment]
@arijitde92 go ahead.
